### ISSUE TITLE
Helm3: Upgrade golang to 1.13

### ISSUE THREAD

**Issue by karuppiah7890** [helm/helm#6380]
Output of `helm version`:
```
version.BuildInfo{Version:"v3.0.0-beta.3", GitCommit:"5cb923eecbe80d1ad76399aee234717c11931d9a", GitTreeState:"clean", GoVersion:"go1.12.9"}
```

This is an issue similar to #6379 . @ialidzhikov would you be interested to take this up too?

**Comment by Aisuko** [comment]
Hi, I'm not sure why to upgrade to golang to 1.13. In fact, this is not an issue right?

**Comment by bacongobbler** [comment]
There are several bug fixes and enhancements in 1.13 that may be useful to chart devwlopers, including the new `slice` function available in text/template. See https://golang.org/doc/go1.13.

**Comment by karuppiah7890** [comment]
@Aisuko I just thought why not keep it up to update as we would keep any other software to get features and of course bug fixes 😅 

Thanks @bacongobbler for putting some solid reasoning to it! 😄

**Comment by karuppiah7890** [comment]
Raised a PR for the same 😄

**Comment by Aisuko** [comment]
Oh, I see, thanks for your reply. Hi, @karuppiah7890  I believe that add @bacongobbler reply into these type of issues could make more sense.

**Comment by karuppiah7890** [comment]
For some reason GitHub automatic issue closing is not happening even though PR mentioned `Fixes` keywords. Encountering this the second time 🤔 

Anyways, this issue can be closed as PR #6392 has been merged ! 😄
